Adding some details for HistorySaveStyle (#10481)
MicrosoftDocs/PowerShell-Docs: reference/7.2/PSReadLine/Set-PSReadLineOption.md

- **SaveAtExit**: Append history file when PowerShell exits.
- **SaveNothing**: Don't use a history file.

> > [!NOTE]
> > If you set `HistorySaveStyle` to `SaveNothing` in a session and then later set `SaveIncrementally`, 
> > the commands previously run in the same session will be saved to history. 

Reviewer: ```suggestion
> [!NOTE]
> If you set **HistorySaveStyle** to `SaveNothing` and then set it to `SaveIncrementally` later in
> the same session, PSReadLine saves all the commands previously run in the session.
```
